Task_TheFlexGame/index.css

```diff
@@ -1 +1,61 @@
 /* Start coding below for FLEX GAME TASK*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.header {
+  background-color: red;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  padding-top: 10px;
+  height: 100%;
```

Comment:
The flex property has a value of 1 1 0, which may not be necessary depending on the desired behavior.